Duplicati/Library/Backend/AzureBlob/AzureBlobBackend.cs

```diff
@@ -195,11 +201,85 @@ public string Description
         public string[] DNSName
         {
             get { return _azureBlob.DnsNames; }
-        }
+        }
+
 
         public void Test()
-        {
-            this.TestList();
+        {
+            // Test enumeration
+            var entriesEnumerator = _azureBlob.ListContainerEntries().GetEnumerator();
+            try
+            {
+                for (int i = 0; i < TEST_FILE_ENUM_COUNT && entriesEnumerator.MoveNext(); i++)
+                {
+                    var fileEntry = entriesEnumerator.Current;
+                }
+            }
+            finally
+            {
+                entriesEnumerator.Dispose();
+            }

```

Comment:
Is there a reason to prefer `try-finally` over a simpler `using` statement?
```
using (var entriesEnumerator = _azureBlob.ListContainerEntries().GetEnumerator())
{
    for (int i = 0; i < TEST_FILE_ENUM_COUNT && entriesEnumerator.MoveNext(); i++)
    {
        var fileEntry = entriesEnumerator.Current;
    }
}
```

Reply:
No good reason here. I used the same approach as in `AzureBlobWrapper.ListContainerEntries()` by inertia.